pyV2DL3/eventdisplay/IrfInterpolator.py

```diff
@@ -66,13 +66,20 @@ def __load_irf(self):
                 irf_axes[i] = np.concatenate(
                     (axis.flatten(), axis.flatten() + 0.01), axis=None
                 )
-        self.irf_data = irf_data
+
+        irf_axes[-2] = np.cos(np.radians(irf_axes[-2]))[::-1]
+        logging.info(irf_axes[-2])
```

Comment:
Not sure what this logging statement does? A printout of values without any additional information is possibly hard to understand.